### PR TITLE
visualstates: 0.1.1-0 in 'kinetic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by okanasik**
Increasing version of package(s) in repository `visualstates` to `0.1.1-0`:

- upstream repository: https://github.com/JdeRobot/VisualStates.git
- release repository: https://github.com/JdeRobot/VisualStates-release.git
- distro file: `kinetic/distribution.yaml`
- bloom version: `0.6.6`
- previous version for package: `null`

## visualstates

```
* set the version of the ros package as 0.1.0
* Merge pull request #55 <https://github.com/JdeRobot/VisualStates/issues/55> from JdeRobot/ros-package
  Generated ROS package dependencies are fixed. fixes #37 <https://github.com/JdeRobot/VisualStates/issues/37>
* cpp ros package can use python run time
* The generated code does not have runtime but depends on the visualstates package itself
* config dialog supports only ROS communication interface
* Merge pull request #40 <https://github.com/JdeRobot/VisualStates/issues/40> from JdeRobot/none-config-fix
  if config is none create it as jderobotconfig and fixes #39 <https://github.com/JdeRobot/VisualStates/issues/39>
* if config is none create it as jderobotconfig
* Merge pull request #36 <https://github.com/JdeRobot/VisualStates/issues/36> from JdeRobot/ros-package
  VisualState tool is redesigned as a ROS package
* ros package of visualstates merging with the final code structure and improvements from master
* fixed remaining empty  merge characters in the source code
* merge from codegen-bug pull request of pushkal
* merge from pushkal's external ros package pull request
* restructure the project for ros
* format commands as source code
* change project directory to have everything under src directory
* Merge pull request #21 <https://github.com/JdeRobot/VisualStates/issues/21> from pushkalkatara/patch-1
  have quotation around path to handle path with with space
* Permissions to script in folders with space
  https://github.com/TheRoboticsClub/colab-gsoc2018-PushkalKatara/issues/8
  Solves #8 <https://github.com/JdeRobot/VisualStates/issues/8>
* Merge pull request #19 <https://github.com/JdeRobot/VisualStates/issues/19> from JdeRobot/subautomata-remove-fix
  Fixing subautomata removal bug
* get the parent node from treemodel
* ignore precompiled python files pyc
* Merge pull request #13 <https://github.com/JdeRobot/VisualStates/issues/13> from JdeRobot/generated_code_dependencies
  Correcting installation path for standalone installation
* add installation directives to README file
* check ROS message directory
* fix the cmake installation path
* Merge pull request #9 <https://github.com/JdeRobot/VisualStates/issues/9> from JdeRobot/standalone_visualstates
  Fixing cmake paths
* fixing cmake paths
* Merge pull request #8 <https://github.com/JdeRobot/VisualStates/issues/8> from JdeRobot/standalone_visualstates
  Migration of VisualStates tool to its own repository
* added examples and updated installation cmake to copy examples under share folder
* ignore .idea directory
* apply small changes for having standalone visualstates
* Merge branch 'master' of github.com:JdeRobot/VisualStates
* first commit
* Solved minor bug in visualStates installation
* remove the sample folder inside visualStates, old examples will reside in src/examples/visualStates
* visualStates_py renamed to visualStates
* remove old visualstates
* #993 <https://github.com/JdeRobot/VisualStates/issues/993> refactored JderobotConfig.cmake included cmake wrappers
* Merge branch 'global/build/792-fix-warnings' of https://github.com/lr-morales/JdeRobot into lr-morales-global/build/792-fix-warnings
* class name change is fixed
* merge from master
* to be able to access cmakevar.hpp add build directory to include directories
* remove the generated file to prevent misunderstanding
* install glade file to general glade path, get cmake variables through a c++ class, fix broken library path after moving install prefix to /opt/jderobot
* merge from jderobot_comm
* [Issue #792 <https://github.com/JdeRobot/VisualStates/issues/792>] Removes compilation warnings from tool visualStates
* [issue #890 <https://github.com/JdeRobot/VisualStates/issues/890>] Solved installation bug in visualStates
* [issue #879 <https://github.com/JdeRobot/VisualStates/issues/879>] solved bug in a visualStates file install
* [issue #879 <https://github.com/JdeRobot/VisualStates/issues/879>] Solved bug in some libraries installation
* [issue #879 <https://github.com/JdeRobot/VisualStates/issues/879>] Updated visualStates.cpp bug with getinterfaces.sh location
* [issue #879 <https://github.com/JdeRobot/VisualStates/issues/879>] updated visualStates.cpp for installation in /opt/jderobot
* [issue #879 <https://github.com/JdeRobot/VisualStates/issues/879>] moved all installation of libs to /opt/jderobot/libs instead of /opt/jderobot/libs/jderobot/
* [issue #879 <https://github.com/JdeRobot/VisualStates/issues/879>] moved all installations to /opt/jderobot
* change name visualHFSM in variable and class names to visualStates
* visualHFSM name is changed to visualStates name in the source code
* Contributors: AAAAAAAAA, Aitor Martinez, Aitor Martínez Fernández, Francisco Pérez, Francisco Rivas, Jose Maria Canas Plaza, Luis Roberto Morales Iglesias, Okan Aşık, Pushkal Katara
```

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
travis is failing because the source and doc `version` need to be a branch and not a tag.

**Comment by okanasik** [comment]
@mikaelarguedas I have followed the bloom tutorial here: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease The tags are created automatically by the bloom tool. Is there a missing information in the tutorial or I miss something. Since tag and branch names collides, I need to delete the tag and create branch with the same name, but this is like undoing the operation done by bloom.

**Comment by mikaelarguedas** [comment]
I see where the confusion comes from.
The source and doc entries are the branches that the ROS buildfarm can compare and run CI or build documentation when new commits are pushed. So they need to be the branches you use in your source repository for developing the code of that package for kinetic. In your case my guess is that it is the `master` branch of https://github.com/JdeRobot/VisualStates. If it;s the case you should specify `master` and not `1.0.0` in the source and doc sections.

There is no need to delete tags or create new branches, just update the name on the branch on this PR.

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -13215,6 +13215,22 @@ repositories:
...
+      type: git
+      url: https://github.com/JdeRobot/VisualStates.git
+      version: 0.1.1
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/JdeRobot/VisualStates-release.git
+      version: 0.1.1-0
+    source:
+      test_pull_requests: true
```
Comment (mikaelarguedas) [inline kinetic/distribution.yaml]: Just checking: Have you followed the [instructions prompted by bloom](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.

Comment (okanasik) [reply kinetic/distribution.yaml]: I will remove test_pull_requests line, since we do not have any test currently.

**Review by mikaelarguedas — DISMISSED** [review]
lgtm, thanks @okanasik for iterating

**Comment by mikaelarguedas** [comment]
Following our [naming conventions](http://wiki.ros.org/ROS/Patterns/Conventions) I recommend making the name of the package more explicit about what it does as well as using the the `lowercase_with_underscores` format.

**Comment by okanasik** [comment]
visualstates is the name of the tool and actually we have been using that name for about 1 year. It is like brand name such as smach. However, maybe we can rename the package with underscore such as `visual_states`. Also sorry for the late reply, I thought the pull request the closed.

**Comment by mikaelarguedas** [comment]
@okanasik sounds good, thanks for the update

I'll leave it up to @dirk-thomas or @tfoote to give additional feedback on this.

**Comment by dirk-thomas** [comment]
I am fine with the package name as is.

**Pull request merged**